### PR TITLE
Idleruns

### PR DESCRIPTION
**Pull request by corpaul**
experiments for 1H, 24H and 1 week idle runs (without systemtap)

### CODE REVIEW

**Comment by whirm** [comment]
Could you enable both the memory profiler and the manhole? We also need to add a passwd file to make manhole work:

```
tribler::1:1::/tmp/:/bin/sh
```

**Comment by corpaul** [comment]
I enabled the profiler and manhole and set the dump times for each experiment. the passwd file will be handled by the meliae pull request

File: experiments/tribler_idle_run/1day.conf [hunk 1]
```diff
@@ -0,0 +1,25 @@
+experiment_name = "TriblerIdleRun24hr"
+workspace_dir = "./"
```
Comment (whirm) [inline experiments/tribler_idle_run/1day.conf]: This is the default workspace value, you can remove config options with default values so they will change if we update gumby.

File: experiments/tribler_idle_run/1day.conf [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+#Custom variable to control the experiment execution time in seconds
+TRIBLER_EXECUTION_TIME = 86400
+
+# memory profiler stuff
+PROFILE_MEMORY = True
+PROFILE_MEMORY_INTERVAL = 3600
+PROFILE_MEMORY_PID_MODULO = 0
+MANHOLE_ENABLE = True
+MANHOLE_PORT = 2323
```
Comment (whirm) [inline experiments/tribler_idle_run/1day.conf]: the ports in each experiment should be different in case they get executed in parallel on the same machine

**Comment by corpaul** [comment]
fixed

File: experiments/tribler_idle_run/1day.conf [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# Enable use of virtualenv when running on the local machine (will be always used on the DAS4)
+use_local_venv = False
+
+#Custom variable to control the experiment execution time in seconds
+TRIBLER_EXECUTION_TIME = 86400
+
+# memory profiler stuff
+PROFILE_MEMORY = True
+PROFILE_MEMORY_INTERVAL = 3600
+PROFILE_MEMORY_PID_MODULO = 0
```
Comment (whirm) [inline experiments/tribler_idle_run/1day.conf]: This is also a default value, the rest looks great

**Comment by corpaul** [comment]
oops, forgot about that one, fixed

**Pull request merged**